### PR TITLE
Ei 362 handle rejected case statuses

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Domain/Accounts/LegalEntityVrfCaseStatus.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿
+namespace SFA.DAS.EmployerIncentives.Domain.Accounts
+{
+    public static class LegalEntityVrfCaseStatus
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Domain/Accounts/LegalEntityVrfCaseStatus.cs:4]: Do we need the LegalEntity bit in the name?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Domain/Accounts/LegalEntityVrfCaseStatus.cs:4]: It's the VRF case status for a given legal entity so thought that naming would be an indication of scope

File: src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationQueryRepository.cs [hunk 2]
```diff
@@ -66,6 +67,14 @@ private static IncentiveApplicationApprenticeshipDto MapToApprenticeshipDto(Ince
...
         private static bool MapBankDetailsRequired(string vrfCaseStatus)
         {
+            if (vrfCaseStatus != null && 
+                  (vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedDataValidation, StringComparison.InvariantCultureIgnoreCase)
+                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVer1, StringComparison.InvariantCultureIgnoreCase)
+                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVerification, StringComparison.InvariantCultureIgnoreCase)))
+            {
+                return true;
+            }
+
```
Comment (Pauljgraham) [inline src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationQueryRepository.cs:77]: would it read a little clearer if we just returned

return (String.IsNullOrWhiteSpace(vrfCaseStatus) ||                (vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedDataValidation, StringComparison.InvariantCultureIgnoreCase)
                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVer1, StringComparison.InvariantCultureIgnoreCase)
                || vrfCaseStatus.Equals(LegalEntityVrfCaseStatus.RejectedVerification, StringComparison.InvariantCultureIgnoreCase)));

**Review by Pauljgraham — COMMENTED** [review]
one tiny comment

**Review by Pauljgraham — APPROVED** [review]

**Pull request merged**